quarkus/extensions/kogito-quarkus-processes-extension/kogito-quarkus-processes-deployment/pom.xml

```
@@ -12,6 +12,10 @@
   <artifactId>kogito-quarkus-processes-deployment</artifactId>
   <name>Kogito :: Quarkus Processes Extension :: Deployment</name>
 
+  <properties>
+    <data-index-ephemeral.image>quay.io/kiegroup/kogito-data-index-ephemeral</data-index-ephemeral.image>
```

i would not set the full path of the image in the property. 
First it is hard to use on command line and second we could invite users/devs to completely overwrite it with no check? Just checking
```suggestion
    <data-index.registry>quay.io</data-index.registry>
    <data-index-ephemeral.image>{data-index.registry}/kiegroup/kogito-data-index-ephemeral</data-index-ephemeral.image>
```

---

@mareknovotny this is consumed during build time for Maven filtering so users can't really change that when using the extension. As for splitting image and registry, I think that's probably unnecessary, as it adds complexity to deal with for instance not providing a repository. It just relies on Docker image naming convention.

---

@spolti that's why I'm keeping a single String parameter there, so during productization build, it only needs to set _-Ddata-index-ephemeral.image=new image name_, and that image will be used by default in the generated artefacts.